database/transaction.rs

```diff
@@ -72,6 +76,7 @@ impl<D: DurabilityClient> TransactionRead<D> {
             query_manager,
             database,
             transaction_options,
+            profile: TransactionProfile::new(tracing::enabled!(Level::TRACE))
```

Comment:
To enable transaction profiles we have to set //database to TRACE. Now only containing Commit, but we could also add Open (causality await is there, for example, and that could be expensive in concurrent scenarios... and we'd want to find that out)